deployment/helm/ditto/README.md

```diff
@@ -1,49 +1,137 @@
-# Eclipse Ditto :: Helm
+# Eclipse Ditto
 
-The Ditto Helm chart sources are managed here.
+## Introduction
 
-## Install Ditto via Helm Chart
+[Eclipse Ditto™](https://www.eclipse.org/ditto/) is a technology in the IoT implementing a software pattern
+called “digital twins”. A digital twin is a virtual, cloud based, representation of his real world counterpart
+(real world “Things”, e.g. devices like sensors, smart heating, connected cars, smart grids, EV charging stations, …).
 
-To install the chart with the release name eclipse-ditto, run the following commands:
+This chart uses `eclipse/ditto-XXX` containers to run Ditto inside Kubernetes.
 
-```shell script
-helm repo add eclipse-iot https://www.eclipse.org/packages/charts/
-helm repo update
-helm install eclipse-ditto eclipse-iot/ditto
+## Prerequisites
+
+Installing Ditto using the chart requires the Helm tool to be installed as described on the
+[IoT Packages chart repository prerequisites](https://www.eclipse.org/packages/prereqs/) page.
+
+TL;DR:
+
+* have a correctly configured [`kubectl`](https://kubernetes.io/docs/tasks/tools/#kubectl) (either against a local or remote k8s cluster)
+* have [Helm installed](https://helm.sh/docs/intro/)
+
+The Helm chart is being tested to successfully install on the three most recent Kubernetes versions.
```

Comment:
The Helm chart is being tested to successfully install on the ~~**three**~~ most recent Kubernetes versions.